docs/study-def-expectations.md

```diff
@@ -5,7 +5,7 @@ OpenSAFELY requires you to define *expectations* in your study definition: these
 ## Defining `return_expectations`
 
 Every variable in a study definition must have a `return_expectations` argument defined (with the exception of the `population` variable).
-This defines the general shape or distribution of the variables in the dummy data used for developing the code.  It is currently relatively unsophisticated; each variable is generated independently of all others. This is sufficient for testing that it is possible to run your study from start to finish in most cases, but sometimes not. You can find (and contribute to!) discussions on improving the [dummy data framework](https://github.com/opensafely/cohort-extractor/issues/221).
+This defines the general shape or distribution of the variables in the dummy data used for developing the code.  It is currently relatively unsophisticated; each variable is generated independently of all others. This is sufficient for testing in that it is possible to run your study from start to finish in most cases, but sometimes not. You can find (and contribute to!) discussions on improving the [dummy data framework](https://github.com/opensafely/cohort-extractor/issues/221).
```

Comment:
Not sure why "in" has been added. The meaning is "In most cases, dummy data is good enough to test that it is possible to run your study from start to finish" perhaps this phrasing is better all round?

Reply:
Ah yes - the sentence just didn't read properly in my head.  I think the alternative phrasing is clearer.